src/angular/controllers/ctrl-widget-catalog.js

```diff
@@ -0,0 +1,225 @@
+import toSnakeCase from 'js-snakecase'
```

Comment:
Since the widget catalog and details pages have been completely rewritten in React, this file is no longer necessary. I'll make a separate commit to remove it.